test/Sentinel1MetadataTest.jl

```diff
@@ -0,0 +1,180 @@
+"""
+unitests for the sentinel-1 metadata
+
+    test
+        1)MetaDataSentinel1
+        2)loading data 
+        3)the different sub structures.
+    
+"""
+
+
+#include("../src/separateLater/Sentinel1/Metadata/Sentinel1Metadata.jl")
+#import .Sentinel1Metadata
+
+
+
+#############################################
+########### test for MetaDataSentinel1 ###########
+#############################################
+
+function MetaDataSentinel1Test()
+    slcMetadata = Sentinel1.MetaDataSentinel1(xmlFile)
+    checkStructures = isdefined(slcMetadata, :header) && isdefined(slcMetadata, :product) && isdefined(slcMetadata, :image) && isdefined(slcMetadata, :swath) && isdefined(slcMetadata, :burstsInfo) && isdefined(slcMetadata, :geolocation)
+    if !checkStructures
+        println("Error in MetaDataSentinel1Test")
+    end
+    return checkStructures
+end
+
+
+#############################################
+########### test for loading data ###########
+#############################################
+
+
+
+function readXmlTest()
+    ## Assert
+    isXML = endswith(xmlFile, ".xml")
+    ## Debug
+    if !isXML
+        println("Input is not .xml format")
+        println(isXML)
+    end
+
+
+    # can the file be read
+    if isXML == true
+        ## Act
+        metaDict = Sentinel1.getDictofXml(xmlFile)
+        ## Assert
+        readXMLcheck = metaDict != nothing
+        ## Debug
+        if !readXMLcheck
+            println("Can't load XML file. Error in getDictofXml() ")
+            println(readXMLcheck)
+        end
+        return isXML && readXMLcheck
+    else
+        return isXML
```

Comment:
Replace this line with `return false` to make it easier to read

Reply:
Done